service/api/handler.go

```diff
@@ -0,0 +1,54 @@
+package api
+
+import (
+	"encoding/json"
+	"github.com/gin-gonic/gin"
+	"github.com/xdblab/xdb/gen/xdbapis"
+	"github.com/xdblab/xdb/service/common/config"
+	"github.com/xdblab/xdb/service/common/log"
+	"github.com/xdblab/xdb/service/common/log/tag"
+	"net/http"
+)
+
+type handler struct {
+	config config.Config
+	logger log.Logger
+}
+
+func newHandler(config config.Config, logger log.Logger) *handler {
+	return &handler{
+		config: config,
+		logger: logger,
+	}
+}
+
+func (h *handler) ApiV1ProcessStartPost(c *gin.Context) {
+	var req xdbapis.ObjectExecutionStartRequest
```

Comment:
Might rename the request to be `ProcessExecutionStartRequest` ?

Reply:
Yeah we need to fix the api repo. 

Also, I realized that moving the gen file in the api repo so that this server repo and Golang SDK repo will look cleaner. I saw some other projects done that recently